scripts/make-packages.sh

```diff
@@ -174,25 +174,63 @@ if [ "$OS" == "linux" ]; then
   if [ -e "$ARTIFACTS/libmsquic.lttng.${LIBEXT}.${VER_MAJOR}.${VER_MINOR}.${VER_PATCH}" ]; then
      FILES="${FILES} ${ARTIFACTS}/libmsquic.lttng.${LIBEXT}.${VER_MAJOR}.${VER_MINOR}.${VER_PATCH}=/usr/${LIBDIR}/libmsquic.lttng.${LIBEXT}.${VER_MAJOR}.${VER_MINOR}.${VER_PATCH}"
   fi
-  fpm \
-    --force \
-    --input-type dir \
-    --output-type deb \
-    --architecture ${PKGARCH} \
-    --name ${NAME} \
-    --provides ${NAME} \
-    --conflicts ${CONFLICTS} \
-    --depends "libssl${TLSVERSION}" \
-    --depends "libnuma1" \
-    --version ${VER_MAJOR}.${VER_MINOR}.${VER_PATCH} \
-    --description "${DESCRIPTION}" \
-    --vendor "${VENDOR}" \
-    --maintainer "${MAINTAINER}" \
-    --package "${OUTPUT}" \
-    --license MIT \
-    --url https://github.com/microsoft/msquic \
-    --log error \
-    ${FILES}
+
+  DONE_DEBIAN=0
+  if [ -e /etc/os-release ]; then
+    source /etc/os-release
+    # Ubuntu 24.04 support xdp
+    if [ "$ID" == "ubuntu" ] && [ "$VERSION_ID" == "24.04" ]; then
```

Comment:
This is a problem... we only build on ubuntu 24.04, but publish for the other distros. We aren't building a version of 24.04 **without** XDP for those other distros. We need to, or they will break.

Reply:
Should this rather be a script argument, same way as openssl 1.1 vs 3 is?

Reply:
This should be a script argument. The environment where we publish packages isn't necessarily the environment where the packages are built. It should never be. We build packages in a different pipeline and reference them in this publishing pipeline.

Reply:
Also, we should only publish ubuntu 24.04 packages to ubuntu 24.04 repo. The other distros that use openssl3 should use packages built on 22.04 and that use openssl should use packages built on 20.04, i.e. same as before.